### PR TITLE
fix(docs): Replace registry with GUI instructions

### PR DESCRIPTION
**Pull request by CarlosNihelton**
Team agreed that the GUI fits best for the user's first contact with the tool.

### CODE REVIEW

File: docs/tutorial/index.md [hunk 1]
```diff
@@ -354,10 +354,16 @@ ssl_public_key = C:\Users\<YOUR_WINDOWS_USER_NAME>\landscape_server.pem
...
-![Registry Editor with the Ubuntu Pro configuration](./assets/regedit-config.png)
 
+Then load that file using the "Custom Configuration" part of the the screen, as shown below:
 
-Done! 
+![Loading Landscape custom config](./assets/loading-custom-landscape-config.png)
+
+Click on the "Continue" button. You'll see a status screen confirming the configuration is complete.
+
+![Configuration is complete](image.png)
```
Comment (EduardGomezEscandell) [inline docs/tutorial/index.md:364]:
```suggestion
![Configuration is complete](./assets/status-complete.png)
```
I think

File: docs/tutorial/index.md [hunk 2]
```diff
@@ -321,22 +321,22 @@ Time to install UP4W! Click on [this link](https://www.microsoft.com/store/produ
...
+### Configure UP4W for Ubuntu Pro and Landscape
 
-In the left pane, navigate to `HKEY_CURRENT_USER\Software\`.
+In the UP4W GUI click in the label "Already have a token?" to expand the Ubuntu Pro token input field.
 
-Create a new key named `Canonical` and, inside of it, create another key named `UbuntuPro`.
+![UP4W GUI main screen](./assets/up4w_gui.png)
 
-Inside the newly created `UbuntuPro` key, add a string value named `UbuntuProToken` and, in its value data field, paste your Ubuntu Pro token. (The token you retrieved from your Ubuntu Pro dashboard during [Setup](ref::ensure-ubuntu-pro)).
+Paste your token retrieved from your Ubuntu Pro dashboard during [Setup](ref::ensure-ubuntu-pro) and click on the "Confirm" button (the button becomes green when there is a valid token in the input field). The app will then show the Landscape configuration screen.
```
Comment (EduardGomezEscandell) [inline docs/tutorial/index.md:337]: I know this is not new, but the way I did most of these labels was (folder::name), so here it'd be `tut::ensure-ubuntu-pro`, as `ref` was for links to the reference.

No need to change it, but just so you know 😅

**Review by EduardGomezEscandell — CHANGES_REQUESTED** [review]
Just the broken link, everything else looks good

File: docs/tutorial/index.md (hunk 1 above)
Comment (EduardGomezEscandell) [reply docs/tutorial/index.md:364]: https://readthedocs.com/projects/canonical-ubuntu-pro-for-wsl/builds/2112015/

File: docs/tutorial/index.md (hunk 2 above)
Comment (CarlosNihelton) [reply docs/tutorial/index.md:337]: I like that suggestion.

**Review by EduardGomezEscandell — APPROVED** [review]
Failing test is a false positive 👍

**Pull request merged**